content/en/blog/_posts/2022-04-19-csi-nodeexpandsecret.md

```diff
@@ -0,0 +1,197 @@
+---
+layout: blog
+title: "Kubernetes 1.25: Use Secrets for Node-Driven Expansion of CSI Volumes."
+date: 2022-07-26T10:00:00-08:00
```

Comment:
The release comms team can assign an actual publication date, possibly just a provisional one for now.

Reply:
Yeah, put it as 26th Aug for now.